Makefile

```diff
@@ -332,6 +332,26 @@ else
 endif
 	docker tag antrea/antrea-ubuntu:$(DOCKER_IMG_VERSION) antrea/antrea-ubuntu
 
+.PHONY: build-ubuntu-controller
+build-ubuntu-controller:
```

Comment:
Better to rename this to `build-controller-ubuntu` to keep it consistent with the final image name.